apps/casdoor/1.774.0/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   casdoor:
-    image: casbin/casdoor:v1.774.0
+    image: casbin/casdoor:v1.776.0
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
The only difference between the two lines is that one specifies "v1.774.0" while the other specifies "v1.776.0". This would not affect the functionality of the service but it is good practice to use the most recent version available if it's compatible with your application setup.